build-tools/xa-prep-tasks/xa-prep-tasks.targets

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask AssemblyFile="$(OutputPath)xa-prep-tasks.dll"   TaskName="Xamarin.Android.BuildTools.PrepTasks.DownloadUri" />
+  <UsingTask AssemblyFile="$(OutputPath)xa-prep-tasks.dll"   TaskName="Xamarin.Android.BuildTools.PrepTasks.SystemUnzip" />
+  <Import Project="..\bundle\bundle-path.targets" />
+  <PropertyGroup>
+    <_AzureBaseUri>https://xamjenkinsartifact.blob.core.windows.net/xamarin-android/xamarin-android/bin/</_AzureBaseUri>
+  </PropertyGroup>
+  <Target Name="_GetBundleOutputPath"
+      DependsOnTargets="GetBundleFileName">
+    <PropertyGroup>
+      <_Bundle>$(OutputPath)$(XABundleFileName)</_Bundle>
```

Comment:
Changed to `$(_BundlePath)`, as it's not a full path (`$(OutputPath)` is relative, by default), and it's not an output path (it's written to and read from).